src/main/java/org/isf/patient/service/PatientIoOperations.java

```diff
@@ -37,20 +37,10 @@
 import org.springframework.transaction.annotation.Transactional;
 
 /**
- * ------------------------------------------
- * PatientIoOperations - dB operations for the patient entity
- * -----------------------------------------
- * modification history
- * 05/05/2005 - giacomo  - first beta version
- * 03/11/2006 - ross - added toString method. Gestione apici per
- * nome, cognome, citta', indirizzo e note
- * 11/08/2008 - alessandro - added father & mother's names
- * 26/08/2008 - claudio    - added birth date
- * modified age
- * 01/01/2009 - Fabrizio   - changed the calls to PAT_AGE fields to
- * return again an int type
- * 03/12/2009 - Alex       - added method for merge two patients history
- * ------------------------------------------
+ * ------------------------------------------ PatientIoOperations - dB operations for the patient entity ----------------------------------------- modification
+ * history 05/05/2005 - giacomo - first beta version 03/11/2006 - ross - added toString method. Gestione apici per nome, cognome, citta', indirizzo e note
+ * 11/08/2008 - alessandro - added father & mother's names 26/08/2008 - claudio - added birth date modified age 01/01/2009 - Fabrizio - changed the calls to
+ * PAT_AGE fields to return again an int type 03/12/2009 - Alex - added method for merge two patients history ------------------------------------------
```

Comment:
no need to reformat all javadoc, especially those with a specific layout

Reply:
corrected